package.json

```diff
@@ -6,11 +6,14 @@
   "author": "Blake Guilloud <blake.guilloud@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "jest": "^21.2.1",
-    "eslint": "^4.9.0"
+    "eslint": "^4.9.0",
+    "jest": "^21.2.1"
   },
   "scripts": {
     "test": "jest",
     "lint": "eslint ./lib ./test"
+  },
+  "dependencies": {
```

Comment:
If you could run `yarn remove ganon` and push up the resolved lock file + package.json, i would appreciate it!